ext/scripts/install_scripts/install_via_pip.pl

```diff
@@ -102,9 +105,18 @@ sub replace_missing_version_for_urls{
     @rendered_line_transformation_functions = (\&replace_missing_version_for_urls);
 }
 
-my $cmd = 
+my $cmd = '';
+
+if($ancestor_pip_package_root_path eq '') {
```

Comment:
I hope you tested that you get an empty string and not `undef` from the command line parser?

Reply:
Yes